src/git/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Use a Python image with uv pre-installed
+FROM ghcr.io/astral-sh/uv:python3.12-bookworm-slim AS uv
```

Comment:
In the longrun we should figure out what it takes to run this on alpine. There is a `ghcr.io/astral-sh/uv:python3.12-alpine`, but we might need to be careful that uv correctly uses system python in that case. Maybe we want to follow up on this PR, but it's not blocking.

Reply:
Okay, maybe let's do this in the form of alpine support for python users who want to create docker mcp servers.